tools/setup.sh

```diff
@@ -18,7 +18,7 @@ fetch_docker_repos() {
     fi
 
     if [[ ! -d ${BASEDIR}"/repos/kong-docker" ]]; then
-        git clone https://github.com/Kong/docker-kong.git ${BASEDIR}/repos/kong-docker --depth=1
+        git clone -b 2.4.0 https://github.com/Kong/docker-kong.git ${BASEDIR}/repos/kong-docker
```

Comment:
Should we find a way to support the latest version (2.5.1) of docker-kong? Also, the `--depth=1` can be kept.

Reply:
done